src/config/process.ts

```diff
@@ -11,7 +11,10 @@ type TwilioConfig = {
 export const GALOY_API_PORT = process.env.GALOY_API_PORT || 4012
 export const GALOY_ADMIN_PORT = process.env.GALOY_ADMIN_PORT || 4001
 
-export const BRIA_WALLET_NAME = process.env.BRIA_WALLET_NAME || "dev"
+const briaUrl = process.env.BRIA_HOST ?? "localhost"
+const briaPort = process.env.BRIA_PORT ?? "2742"
+export const BRIA_ENDPOINT = `${briaUrl}:${briaPort}`
```

Comment:
please create a function to return all bria configs

Reply:
Done https://github.com/GaloyMoney/galoy/pull/2662/commits/fa660d125254e88c9e2f78a68d47fa1d34a763c7